binding/python/raftify.pyi

```diff
@@ -79,7 +80,7 @@ class Slogger:
     def default() -> "Slogger": ...
     @staticmethod
     def new_file_logger(
-        log_path: str, chan_size: int, rotate_size: int, rotate_keep: int
```

Comment:
Why is `level` int type? 
It seems that after adding the `Level` type to the pyi file, it should be assigned to that type.

Reply:
Additionally, as the number of arguments increases, it seems preferable to make arguments other than `log_path` optional.